tests/std/tests/P1502R1_standard_library_header_units/custom_format.py

```diff
@@ -1,129 +1,221 @@
 # Copyright (c) Microsoft Corporation.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+import json
 import os
+import re
 
 from stl.test.format import STLTestFormat, TestStep
 from stl.test.tests import TestType
 
 
+# Print noisy progress messages that are useful when working on this script.
+noisyProgress = False
+
+
+# P1502R1_standard_library_header_units/test.cpp cites the definition of "importable C++ library headers".
+def getImportableCxxLibraryHeaders():
+    return [
+        'algorithm',
+        'any',
+        'array',
+        'atomic',
+        'barrier',
+        'bit',
+        'bitset',
+        'charconv',
+        'chrono',
+        'codecvt',
+        'compare',
+        'complex',
+        'concepts',
+        'condition_variable',
+        'coroutine',
+        'deque',
+        'exception',
+        'execution',
+        'filesystem',
+        'format',
+        'forward_list',
+        'fstream',
+        'functional',
+        'future',
+        'initializer_list',
+        'iomanip',
+        'ios',
+        'iosfwd',
+        'iostream',
+        'istream',
+        'iterator',
+        'latch',
+        'limits',
+        'list',
+        'locale',
+        'map',
+        'memory_resource',
+        'memory',
+        'mutex',
+        'new',
+        'numbers',
+        'numeric',
+        'optional',
+        'ostream',
+        'queue',
+        'random',
+        'ranges',
+        'ratio',
+        'regex',
+        'scoped_allocator',
+        'semaphore',
+        'set',
+        'shared_mutex',
+        'source_location',
+        'span',
+        'spanstream',
+        'sstream',
+        'stack',
+        'stdexcept',
+        'stop_token',
+        'streambuf',
+        'string_view',
+        'string',
+        'strstream',
+        'syncstream',
+        'system_error',
+        'thread',
+        'tuple',
+        'type_traits',
+        'typeindex',
+        'typeinfo',

```

Comment:
The Standard table's ordering doesn't match Python `sorted()` (or C++ `sort()`) here. The ASCII value of `'_'`, 95, is lower than the ASCII value of lowercase `'i'`, 105.

I would prefer to match ASCIIbetical ordering here, since we apply `sorted()` before using this list.

Reply:
Works for me.